fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordContext.java

```diff
@@ -530,6 +530,24 @@ static void clearMdc(@Nonnull Map<String, String> mdcContext) {
         }
     }
 
+    /**
+     * Get a new {@link FDBDatabaseRunner} that will run contexts similar to this one.
+     * <ul>
+     * <li>Same {@link FDBDatabase database}</li>
+     * <li>Same {@link FDBStoreTimer timer}</li>
+     * <li>Same {@link #getMdcContext() MDC context}</li>
+     * <li>Same {@link FDBDatabase.WeakReadSemantics weak read semantics}</li>
```

Comment:
```suggestion
     * <li>Same {@linkplain FDBDatabase.WeakReadSemantics weak read semantics}</li>
```

Same for the other links in this comment.

Reply:
Fixed.